### PR TITLE
function kill command introduction

### PR DESCRIPTION
**Pull request by dwibedis**
* Introducing `FUNCTION KILL` command

### CODE REVIEW

**Comment by dwibedis** [comment]
ref:- https://github.com/redis/go-redis/issues/2398

File: commands_test.go [hunk 1]
```diff
@@ -6034,6 +6034,17 @@ var _ = Describe("Commands", func() {
 			Expect(len(result)).NotTo(BeZero())
 		})
 	})
+
+	Describe("Functions", func() {
+		It("should FUNCTION KILL", func() {
+			fKill := client.FunctionKill(ctx)
+			Expect(fKill.Err()).Should(Equal("NOTBUSY No scripts in execution right now."))
```
Comment (dwibedis) [inline commands_test.go:6041]:
```suggestion
			Expect(fKill.Err()).Should(Equal("No scripts in execution right now."))
```

**Comment by chayim** [comment]
@elena-kolevska you should see how this ties with what you're doing.

**Comment by monkey92t** [comment]
See #2475, close this PR.

**Pull request closed without merging**